packages/help-center/src/components/help-center-article.scss

```
@@ -111,19 +111,22 @@
 				}
 			}
 
-			p, ol.wp-block-list {
+			p {
 				font-size: $font-body-small;
 				color: var(--color-neutral-80);
 			}
 
-			ul {
+			ul, ol.wp-block-list {
```

```suggestion
			ul, ol {
```
Nit, but it seems safer to target all `ol`s

---

In this case, it would affect the `font-size` for `ol.toc-parent-list` and `ol.toc-child-list` blocks, that's why I preferred to have a more specific selector. I can also rewrite to exclude these two blocks instead (something like this: `ol:not(.toc-parent-list):not(.toc-child-list)`), but it would be harder to maintain since the exclusion list could potentially grow when new `ol` blocks are created.

---

Good catch 👍 Seems like this is the better approach after all, then